src/ast/ddl.rs

```diff
@@ -217,6 +217,14 @@ impl fmt::Display for AlterColumnOperation {
     }
 }
 
+/// For mysql index, here may be there format for index, there patterns are similar。
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum KeyFormat {
```

Comment:
As I stated in another comment, `UNIQUE` is a constraint. Please don't change the current handle structure for it, as this doesn't make sense. You're changing the AST to match only one dialect.

Reply:
If you don't do this. this libaray would never support for mysql create statement. I think we should make it work first, and if someone have a better solution，we can apply it later.

Reply:
I'll try something up, would you mind waiting a day or two?

Reply:
@RainJoe would you mind if I tried a different approach and you took a look in a day or two?

Sorry for asking that, I know that this has been your work, don't want to look like throwing that away

Reply:
Of course not. waitting for your approach.

Reply:
@RainJoe @alamb could you please leave your opinion? https://github.com/sqlparser-rs/sqlparser-rs/pull/665

Reply:
@AugustoFKL Thanks for your work, but it didn't solve my problem, maybe I would just use my fork repository. It's seems hard to consider all possible scenarios.

Reply:
@RainJoe can you give examples of what you intend about this task?

The index usage itself is being parsed... What is the syntax you need?